core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -243,45 +285,52 @@ private static boolean validateMasters(String name) throws InterruptedException
    */
   public static void printHelp(String message) {
     System.err.println(message);
-    System.out.print(USAGE);
+    HelpFormatter formatter = new HelpFormatter();
+    formatter.printHelp(USAGE, null, OPTIONS, null, true);
```

Comment:
(nit) You can probably use the shorter version `formatter.printHelp(USAGE, OPTIONS, true);`

Reply:
Done.